.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
```

Comment:
- [ ] Please good job by adding the background using css as the last reviewer said but it would be better to fix the position of this background as it doesn't match the design.

| Your code | Figma |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/212506171-ab2df64d-9698-4332-a64a-b5d4fe707fbc.png) | ![image](https://user-images.githubusercontent.com/74744787/212506557-eef02b06-9ab3-4c4a-aaca-e42fc22ddebe.png)  |

Reply:
![image](https://user-images.githubusercontent.com/122236545/212771180-ed25d731-acc6-4593-99c1-dfdb1b618a80.png)

It looks as per the template in my browser

Reply:
Nevermind, I got to see what is the problem here